### PR TITLE
Documentation about how to configure a customer infrastructure to use a proxy

### PR DESCRIPTION
**Pull request by afjsilva**
Added documentation about how to configure a customer infrastructure to use a proxy and their docker daemon to respect and use the proxy configuration to.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,3 +106,61 @@ parameters=""
...
 ```
+
+
+## Configure customer infrastructure to use a Proxy
+
+When you want to use a Proxy and configure a infrastructure/machine to pass all the connections through a proxy, some troubles can appear to be possible to communicate with the external network and get the necessary packages and images from the registry. You must set some environment variables on your system, to permit your machine to communicate through your proxy and access the external network to download the essencial packages and others.
+
+Assuming you are using a linux machine (for windows you must try achive the same goal, creating environemnt variables and configuring the docker daemon), you can add this next export commands lines on your */etc/bash.bashrc* (for your bash start with this environment variables) or do you just run. The idea is to add the necessary environment variables to your system.
```
Comment (francisco-maciel) [inline README.md:115]: "Assuming you are using a linux machine (for windows you must try achive the same goal, creating environemnt variables and configuring the docker daemon), you can add this next export commands lines on your */etc/bash.bashrc* (for your bash start with this environment variables) or do you just run. The idea is to add the necessary environment variables to your system."

TO

"For Linux machines, add the following export commands lines on your */etc/bash.bashrc*, ensuring that bash will start with additional environment variables. The goal is to add the necessary environment variables to your system. For Windows machines you can try achive the same goal by creating environemnt variables and configuring the docker daemon

File: README.md [hunk 2]
```diff
@@ -106,3 +106,61 @@ parameters=""
...
+```
+
+Or to set in your environment variables file (*/etc/environment*) below the PATH environment:
+
+```bash
+HTTP_PROXY=http://<<proxyuser:proxypassword>@><proxy.example.com>:<proxyport>
+HTTPS_PROXY=http://<<proxyuser:proxypassword>@><proxy.example.com>:<proxyport>
+```
+
+Depending on the software or system, can be necessary you add the same environment variables in lower case to, this is 'http_proxy' and 'https_proxy'.
```
Comment (francisco-maciel) [inline README.md:129]: it can

File: README.md [hunk 3]
```diff
@@ -106,3 +106,61 @@ parameters=""
...
+
+To define this variables to all your system, you can define on the file */etc/apt/apt.conf* (in some system on another place like */etc/apt/apt.conf.d/{something-proxy}*) the following code:
```
Comment (francisco-maciel) [inline README.md:131]: location such as

**Review by francisco-maciel — CHANGES_REQUESTED** [review]

**Comment by afjsilva** [comment]
@francisco-maciel can you please check :) I applied your suggestions.

**Review by francisco-maciel — APPROVED** [review]

**Pull request merged**